README.md

```diff
@@ -10,7 +10,8 @@ Cloud Native Computing Taiwan User Group is dedicated to combine community of Cl
 - [KKTIX](https://cntug.kktix.cc)
 - [HackMD](https://hackmd.io/@CNTUG)
 - [Meetup 投影片(Slides)](slides/README.md)
-- Cooperative communities.
+- [OCF Donate](https://ocf.neticrm.tw/civicrm/contribute/transact?reset=1&id=29&_ga=2.247282817.443417200.1638513519-757737249.1637652904)
```

Comment:
IMO, it's better to only leave id=29 and delete other arguments to make it shorter.

Reply:
```suggestion
- [OCF Donate](https://ocf.neticrm.tw/civicrm/contribute/transact?id=29)
```
Thank you for your carefulness.
I also agree the query string to leave only what is necessary.
Is the above adjustment content better?